bedrock/firefox/templates/firefox/firstrun/base.html

```diff
@@ -2,12 +2,37 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 
+{% add_lang_files "firefox/new/onboarding" %}
+
 {% extends "firefox/base.html" %}
 
 {% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
 
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{ _('Welcome to Firefox') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+{% block page_desc %}
+  {% if l10n_has_tag('macos_branding') %}
+    {{_('Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows, macOS, Linux, Android and iOS today!')}}
+  {% else %}
+    {{_('Download Mozilla Firefox, a free Web browser. Firefox is created by a global non-profit dedicated to putting individuals in control online. Get Firefox for Windows, Mac OS X, Linux, Android and iOS today!')}}
+  {% endif %}
+{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}{{ _('Chart your own course') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{_('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.')}}{% endblock %}
+
```

Comment:
Any reason to omit the `page_og_url` block here? Seems it would be good to have in the event someone tries to share this URL.

Reply:
Ah, looks like the onboarding template omitted this - will add it back, nice spot